dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/Reloader.scala

```diff
@@ -64,7 +72,11 @@ object Reloader {
     def reload(): Unit
 
     /** URL at which the application is running (if started) */
+    @deprecated("1.5.0", "Use #bindings() instead.")
     def url(): String
+
+    /** List of bindings this server is exposing.*/
+    def bindings(): Seq[DevServerBinding] = DevServerBinding.fromString(url())
```

Comment:
Deprecated `url` in favor of a collection that is also more strongly typed.

Reply:
We don't need to have a default implementation for `bindings()`. This is used in only two places and in both cases we redefine it.